src/CSharp/CodeCracker/Style/PropertyPrivateSetAnalyzer.cs

```diff
@@ -39,7 +39,7 @@ private static void AnalyzeClass(SyntaxNodeAnalysisContext context)
             var invocationExpression = (PropertyDeclarationSyntax)context.Node;
             var semanticModel = context.SemanticModel;
 
-            if (invocationExpression.AccessorList.Accessors.Count == 1) return;
+            if (invocationExpression.AccessorList == null || invocationExpression.AccessorList.Accessors.Count == 1) return;
```

Comment:
I think taht, in this case, you can even shorten it more:

    invocationExpression.AccessorList?.Accessors.Count == 1

If `invocationExpression.AccessorList` is `null`, `invocationExpression.AccessorList?.Accessors.Count` won't be `1`.

I don't have VS2015 here, but I tried this:

    using System;
    using System.Collections.Generic;
    public class C
    {
        public void M()
        {
            Dictionary<string,string> d = null;
            if (d?.Keys.Count == 1)
            {
            }
        }
    }

in [http://tryroslyn.azurewebsites.net/](http://tryroslyn.azurewebsites.net/#K4Zwlgdg5gBAygTxAFwKYFsDcAoUlaIoYB0AwgPYA2lqAxsmORCMQOKoSoBOYtOADsABGlXjFqUAhiBAxS2AN7YYKmIJFiAbuTAATGAFkAFAEplqpaqswAIrwZNJXBAB4UPaABp3+AHwx9AF4YCGBqHGtVMAAzGCNdAH5iAGlUJDJyYAhkGEDggEYzSJVLYoBfcxUKsqAAA=)